cert/lib/cert/runner.rb

```diff
@@ -134,6 +134,19 @@ def certificates
 
     # The kind of certificate we're interested in
     def certificate_type
+      if Cert.config[:type]
```

Comment:
Is it possible for `Cert.config[:type]` to be nil?

Reply:
Yup yup, its optional and takes precedence over the `:development` option which is why we check for it first